lib/src/widgets/home_route.dart

```diff
@@ -35,134 +27,36 @@ class _HomeRouteState extends State<HomeRoute> {
           ButtonBar(
             children: <Widget>[
               IconButton(
-                  icon: const Icon(Icons.filter_list_outlined),
+                  icon: const Icon(Icons.circle_notifications_outlined),
                   onPressed: () => {
-                        //TODO: implement
+                        context.go('/in_dev'),
                       }),
-              IconButton(
```

Comment:
Why was the navigation changed?

Reply:
Inherited Model state hoisting. Floating action button can be re-added.

Reply:
Let's keep the PR scope limited. If this one is about updating dependencies, it should contain only related changes. 
Furthermore, PRs should not revert previous changes if this is not a desired change.

Reply:
I will add back the floating action button

Reply:
Completed. Ready to merge.